src/org/usfirst/frc4904/robot/commands/OverridableCommand.java

```diff
@@ -0,0 +1,30 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+/**
+ * 
+ * @author niksure
+ *         This interface allows commands to easily override other commands
+ */
+public interface OverridableCommand {
+	/**
+	 * sets an override to an static boolean you give it
+	 * 
+	 * @param override
+	 */
+	void setOverride(boolean setValue);
+
+	/**
+	 * returns if override is on or off
+	 * 
+	 * @return
+	 */
+	boolean getOverride();
+
+	/**
+	 * opposite of getOverride
+	 * 
+	 * @return
+	 */
+	boolean getNotOverride();
```

Comment:
Can you add a default implementation for this? This is a new feature in Java 8 and you can see an example in `Nameable.java` (in `standard.custom`).